vizro-core/docs/pages/explanation/faq.md

```diff
@@ -34,7 +34,17 @@ We do not consider frontend changes (such as changing the appearance of a compon
 
 ## Where can I find example dashboards?
 
-Check out our [examples section](../examples/examples.md) which describes the contents of the `examples` folder of the [Vizro GitHub repository](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples).
+For a gallery of examples of Vizro in action, take a look at [vizro.mckinsey.com](https://vizro.mckinsey.com).
+
+Within Vizro's GitHub repository, the [`examples/dev` folder](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/dev) of `vizro-core` contains example code using the pydantic model. If you have any problems running the example code, please [raise an issue](https://github.com/mckinsey/vizro/issues) on the Vizro repository.
+
+!!! note
+
+    The pydantic model should be your preferred method of dashboard configuration, but you can also [define a dashboard with YAML, JSON, or as a Python dictionary](../user-guides/dashboard.md). To guide you, the `examples/dev` folder also contains an alternative `yaml_version` of the gapminder example, which offers the same example as the pydantic model but via YAML configuration.
+
+### Examples from Vizro users
+
+We maintain a separate, curated page of [videos, blog posts, and examples of Vizro usage from our community](your-examples.md).
```

Comment:
```suggestion
We also maintain a separate, curated page of [videos, blog posts, and examples of Vizro usage from our community](your-examples.md).
```

Reply:
Now that we have HuggingFace up and running, these dev examples are really intended only for developers, and they're explained in the contributing guide.

Things which I think might still be worth highlighting here or elsewhere in the documentation, in this PR or ini another one. wdyt @maxschulz-COL?
1. The HuggingFace gallery allows you to look at code as well as the dashboards
2. Swap our HF features dashboard into the YAML version of it and link to that as a complete example of a YAML dashboard in https://vizro.readthedocs.io/en/latest/pages/user-guides/dashboard/
3. Link in https://vizro.readthedocs.io/en/latest/pages/user-guides/run/#jupyter to the Jupyter notebook `dev/example` because I think that's the only Juypyter notebook example we have? Or maybe just delete that example entirely since we never use it and it doesn't add anything to the docs really, since we already have a copy and pastable example there of a simple dashboard

Reply:
1. Agree, where do we want to mention?
2. Sure can do, I'll create a ticket: https://github.com/orgs/McK-Internal/projects/15/views/9
3. I think we need that section, because it allows us to explain the problems with the managers! But we can delete the source code. @stichbury wanna do that quickly?

Wdyt guys?

Reply:
1. I'd probably mention it here, something like this:
```
For a gallery of examples of Vizro in action, take a look at [vizro.mckinsey.com](https://vizro.mckinsey.com). For dashboards in our the [HuggingFace gallery](https://huggingface.co/vizro), you can see the app code by clicking the "Files" tab in the top banner.

We also maintain a separate, curated page of [videos, blog posts, and examples of Vizro usage from our community](your-examples.md).
```

2. Thanks!
3. Agree we want the section _and_ we actually also want the source code because it shows the `jupyter_mode` argument and makes it easy for a user to copy and paste into a Jupyter notebook. Sorry I wasn't clear before. The thing I think we should consider deleting is the `example/dev/app.ipynb` example itself, not the docs.

Reply:
Yes let's delete the code in `example/dev/app.ipynb` but not the docs example.

Reply:
Thanks both, I'll put that in now.